<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheBigWindow</class>
 <widget class="QWidget" name="TheBigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Talisman</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="bigSquare"/>
  </widget>
  <widget class="QPushButton" name="btnRollDie">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll Die</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLeft">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>40</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>counterclockwise</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRight">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>40</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>clockwise</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYes">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>80</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QLabel" name="cross">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Do you want to cross?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNo">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>80</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QLabel" name="crownReached">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>30</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>You have reached the 
 crown of command!</string>
   </property>
  </widget>
  <widget class="QLabel" name="instructions">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>40</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose a character 
 by clicking the card: </string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="instructionBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>801</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="iLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Square Instructions:</string>
   </property>
  </widget>
  <widget class="QLabel" name="movesLeft">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>move 2 more squares</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
